elasticgraph-indexer_autoscaler_lambda/lib/elastic_graph/indexer_autoscaler_lambda/concurrency_scaler.rb

```diff
@@ -59,20 +66,22 @@ def tune_indexer_concurrency(queue_urls:, min_cpu_target:, max_cpu_target:, maxi
               (current_concurrency - (current_concurrency * decrease_factor)).round.tap do |new_concurrency|
                 details_logger.log_decrease(
                   cpu_utilization: cpu_utilization,
+                  min_free_storage: free_storage,
                   current_concurrency: current_concurrency,
                   new_concurrency: new_concurrency
                 )
               end
             else
               details_logger.log_no_change(
                 cpu_utilization: cpu_utilization,
+                min_free_storage: free_storage,
                 current_concurrency: current_concurrency
               )
               current_concurrency
             end
           else
             details_logger.log_reset
-            0
+            MINIMUM_CONCURRENCY
```

Comment:
Why the change from 0 to 2 here?

Also, out of curiosity...now that we're not scaling the SQS trigger, can the min be 0 or 1 instead of 2?

Reply:
The min can't be 0 since we always increase by a factor of the current concurrency so it would never increase from 0 in that case. It _could_ be 1, but the starting value changes the rate at which it increases since it doubles and I thought the rate was pretty good with 2. Even though we clamp the value between the min and max when it's 0 here, it still triggers a lambda update which is why sometimes there's an error in a TF apply that the lambda is in the process of being updated. I think this solves that